app/src/main/java/corp/kairos/adamastor/Util.java

```diff
@@ -13,4 +13,24 @@ public static Object getObjectByIndex(int position, Set set) {
         }
         return null;
     }
+
+
+    public static String getContextNameById(int id) {
```

Comment:
This method should be a part of the Class UserContext. Here it is... _out of context_ 😛